README.md

```diff
@@ -1,8 +1,274 @@
-# RFC
+# Amethyst RFCs
 
-This is the repository where we work and keep track of the all RFC documents related to Amethyst.
-Having this repository allows people to participate in decision making by opening a pull request with a new document.
+[Amethyst RFCs]:
 
-## Creating a new RFC
+Note that this will be actual links once the bot is set up
+- discussing RFCs (should link to all open PRs here)
+- accepted RFCs (can link to the file tree)
+- RFCs waiting for implementation (issues with label="RFC accepted" -label="status: working"
```

Comment:
You seem to be missing a closing parenthesis here :wink:

Reply:
Fixed